countLinesFile.py

```diff
@@ -0,0 +1,5 @@
+rax = open('Hello.txt','r')
+count = 0
+for nat in rax:
+    count = count + 1
+print('Total Lines : ',count)

```

Comment:
Can you put this program to `basics/files.md`, near other code examples of looping over a file, and add some text explaining what this code does?